app/services/tufts/shared_terms_service.rb

```diff
@@ -0,0 +1,12 @@
+module Tufts
```

Comment:
Thinking about my question in the other comment, could this just be:

```ruby
module Tufts
  class Terms
    def self.shared_terms
      [:displays_in, # etc...
      ]
    end
  end
end
```

Then, when needing to use the terms do `Tufts::Terms.shared_terms`?

We could also move the terms array to a frozen constant. Right now we allocate the array from scratch each time the method is called.

```ruby
class Terms
  SHARED_TERMS = [:displays_in, # etc...
                                    ].freeze
  def self.shared_terms
    SHARED_TERMS
  end
end
```